src/js/constants/EventTypes.js

```diff
@@ -1,11 +1,13 @@
 let EventTypes = {};
 [
-  'AUTH_USER_LOGIN_SUCCESS',
+  'APP_STORE_CHANGE',
+  'AUTH_LOGIN_REDIRECT',
   'AUTH_USER_LOGIN_ERROR',
-  'AUTH_USER_LOGOUT_SUCCESS',
+  'AUTH_USER_LOGIN_SUCCESS',
   'AUTH_USER_LOGOUT_ERROR',
-  'AUTH_LOGIN_REDIRECT',
-  'APP_STORE_CHANGE',
+  'AUTH_USER_LOGOUT_SUCCESS',
+  'CHRONOS_JOBS_CHANGE',
```

Comment:
Just introducing ` 'CHRONOS_JOBS_CHANGE'` and `'CHRONOS_JOBS_ERROR'` here. The reset is :abc: